CMakeLists.txt

```diff
@@ -239,6 +239,15 @@ add_compile_definitions(
 
 if (NOT WIN32)
     find_package(SDL2 REQUIRED)
+    # ref: issue-1595
```

Comment:
This fix should go into cmake/FindSDL2.cmake

Reply:
Yes there is no such module right now...
It can be moved there but it may be tricky - it probably should not be `FindSDL2.cmake` otherwise Cmake won't check system-wide one in `/usr/lib/x86_64-linux-gnu/cmake/SDL2/` (but we still need use it to actually get library information)   

May check on that approach a later if it is a blocker

Reply:
I will upload a new revision 😅

Reply:
I agree that we don't need FindSDL2.cmake